pages/common/z.md

```diff
@@ -0,0 +1,27 @@
+# z
+
+> Tracks your most used directories and enable quick jumping using regexes from the CLI.
+
+- Go to a directory that contains "foo":
+
+`z foo`
+
+- Go to a directory that contains "foo" and then "bar":
+
+`z foo bar`
+
+- Go to the highest ranked directory matching foo:
+
+`z -r foo`
+
+- Go to the most recently accessed directory matching foo:
+
+`z -t foo`
+
+- List all directories matching foo (from z datafile):
+
+`z -l foo`
+
+- Remove the current directory from the datafile:
```

Comment:
I think it might be useful to use words that convey the "meaning" of the example rather than copy words from the README verbatim. 

From what I understand, the "datafile" is a file used by the z command to store the directory usages. In that context, it might be better to use the word "datastore" or "database". "datafile" is a bit of an uncommon word, and we want to avoid doubt as much as possible.

I suggest -
```
Remove the current directory from z database:
```

Same for the previous description too.